kahuna/public/js/search/query.html

```diff
@@ -91,7 +91,7 @@
                            name="sort-direction"
                            ng-value="undefined"
                            ng-model="searchQuery.ordering.orderBy">
-                    <label for="sort-direction__desc" class="radio-list__label"
+                    <label tabindex="0" ng-keypress="accesibilityKey($event, 'undefined')" for="sort-direction__desc" class="radio-list__label"
```

Comment:
I think this should be possible without custom js. 

Using https://www.sarasoueidan.com/blog/inclusively-hiding-and-styling-checkboxes-and-radio-buttons/ should allow the checkboxes to be hidden while preserving tab order. (Not using display:none) And then I think https://www.a11ymatters.com/pattern/checkbox/ would help with the focus style, using the `:focus` selector and then using that to get to the label.

Reply:
I have applied the requested changes following the suggested blog post approaches. Shout out to the writers.